app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -163,6 +167,18 @@ class ChatActivity : AppCompatActivity(), IChatView {
             }
             handled
         })
+
+        askSusiMessage.setOnKeyListener(View.OnKeyListener { view, i, keyEvent ->
+            if (i == KeyEvent.KEYCODE_ENTER && enterAsSend && (keyEvent.action == KeyEvent.ACTION_UP || keyEvent.action == KeyEvent.ACTION_DOWN)) {
+                val message = askSusiMessage.text.toString().trim({ it <= ' ' })
```

Comment:
So, are we disallowing sending only spaces as messages?

Reply:
@iamareebjamal Yes! as earlier method did